### DISCUSSION TITLE
Could someone please provide me with qhd icm file?

### DISCUSSION THREAD

**Discussion by pyunjiji** [seerge/g-helper#2190] [Q&A] [answered]
Hello? My laptop was originally an fhd display, but I upgraded the display to qhd. So I only have icm files of fhd in C:\ProgramData\ASUS\GameVisual. If you have a qhd laptop user, could you please provide the C:\ProgramData\ASUS\GameVisual folder?

**Comment by seerge** [comment]
You need to mention full model name at least here so people can assist you

**Reply by seerge** [reply]
@pyunjiji check this build, I have added a button to download ICC profiles if they are not there yet
[GHelper.zip](https://github.com/seerge/g-helper/files/14551004/GHelper.zip)

**Reply by pyunjiji** [reply]
My model name is  FA707RR-HX001. But I need Qhd panel's Profile.  So FA707NU-DS74 Profile is required.

**Reply by IceStormNG** [reply]
It is the same package, there are different profiles in it for the different panels. The package for the ICC Profiles does not depend on the panel but only on the laptop. The package contains all profiles for all panels that ASUS sells with this laptop.

![image](https://github.com/seerge/g-helper/assets/12786283/20f29feb-113e-4d6d-9e8b-c13102bff5ac)

If the panel is reported correctly by the ACPI, GHelper will automatically pick the right profile.

**Reply by pyunjiji** [reply]
@IceStormNG 
Could you please provide the above files?

**Reply by IceStormNG** [reply]
If you downloaded them with GHelper, you can find them in C:\ProgramData\ASUS\GameVisual
That's the same files that GHelper downloads when you click on the download button for Visual styles.

**Reply by pyunjiji** [reply]
@IceStormNG
Yes, I know, but it doesn't work for the above reasons, so I asked you for a file because I was wondering if it would work if I used qhd's file.

But when I checked, the FA707NU-DS74 model was also fhd

![스크린샷 2024-03-14 171217](https://github.com/seerge/g-helper/assets/57628758/e2edfa7a-a836-4adb-8021-59f6f8691f9e)

**Reply by IceStormNG** [reply]
https://dlcdn-rogboxbu1.asus.com/pub/ASUS/APService/Gaming/SYS/ROGS/14456-VAZ4LS-1d798d3873c55e9d10b43a3f2b12cd15.zip

you can download the bundle here from ASUS.

**Reply by n1gq** [reply, marked as answer]
@pyunjiji 
There are tons of qhd rog laptop with different qhd panels. Blindly asking someone else’s GV folder won’t do you any favors.  Identify which qhd panel you’ve got and start there. 

GameVisual (and GHelper) follows a certain rule to apply the correct icm files on different devices. Here’s an example:
![image](https://github.com/seerge/g-helper/assets/163407732/6d811cea-ecf8-43e1-a688-9ca9d311fb2c)
G634JYR = laptop model name (scar 16 4090 2024)
10DE / 8086 = gpu (nvidia / intel)
E509 = panel manufacturer (BOE)
0C24 = panel hardware id (NE160QDM-NM7)

GPU id, panel manufacturer and panel hardware id can be found on windows device manager as shown:
![image](https://github.com/seerge/g-helper/assets/163407732/1a503cad-7ba8-47a5-9404-15170f964243)
![image](https://github.com/seerge/g-helper/assets/163407732/59f86d13-bdf4-4ca3-a6de-7351f4711d6e)

Currently I have scar16 2024 panel installed on zephyrus m16 2022. To apply the correct icc profile through GV (or ghelper), I renamed the scar16 icm file to:
![image](https://github.com/seerge/g-helper/assets/163407732/d7b3672f-a931-4ed7-b419-d0363da1f7d3)
Everything works fine including color gamut switching.
![image](https://github.com/seerge/g-helper/assets/163407732/38a558ba-246f-4ff2-8cf1-4062173a6976)

For your case:

If the panel is from your laptop's other sku (i.e. other screen option), GV/GHelper should identify it and works with the original icm files.

If the panel is from other rog laptop, you have to ask for that specific model's GV folder. Then identify which exact panel you've got and rename the corresponding icm file. It should work after you rename the file correctly.

If the panel is NOT from other rog laptop, you are out of luck with GV/GHelper. Do you own monitor calibration.

**Reply by pyunjiji** [reply]
@n1gq   THANKS! I am using NE173QHM-NZ1 from BOE and the product ID is BOE09F90. I was able to operate the profile correctly using the method you told me.

**Reply by Tozward** [reply]
Hi there! @n1gq I'm swapping to NE160QDM-NM7 panel but on a non-ASUS laptop (which disabled me to utilize Armoury Crate), and I'm in urgent need of exactly the same ICC profiles as yours. Could you plz share the profiles? Would be of great help!

**Comment by ke1ne** [comment]
> @pyunjiji check this build, I have added a button to download ICC profiles if they are not there yet [GHelper.zip](https://github.com/seerge/g-helper/files/14551004/GHelper.zip)

Dude, you're awesome (again)!

**Reply by Aviral1997** [reply]
I want icc files for FA566IU can you help please

**Reply by IceStormNG** [reply]
> I want icc files for FA566IU can you help please

There are no color profiles for this model. Are you sure it supports Game Visual when using Armoury Crate?

**Reply by seerge** [reply]
Based on asus website https://www.asus.com/supportonly/fa566iu/helpdesk_download/ it seem to be some strange regional version of FA506UI :) (at least BIOS section offers BIOS for 506)

**Reply by IceStormNG** [reply]
There is also no color profile for that one ... It looks like some TUF with the cheap 144Hz FHD Panel.  These panels do not even cover sRGB.

**Reply by Aviral1997** [reply]
> > I want icc files for FA566IU can you help please
> 
> There are no color profiles for this model. Are you sure it supports Game Visual when using Armoury Crate?

It supports game visual and it is tuf a 15 model

**Reply by IceStormNG** [reply]
It looks like they do this without a specific color profile on those machines. I cannot find any for this model or a model with a similar display. They all have no profiles for download.
